Mono.Debugging/Mono.Debugging.Evaluation/ObjectValueAdaptor.cs

```diff
@@ -857,7 +857,7 @@ public ValueReference GetMember (EvaluationContext ctx, IObjectSource objectSour
 			return GetMember (ctx, objectSource, GetValueType (ctx, co), co, name);
 		}
 
-		public ValueReference GetMember (EvaluationContext ctx, IObjectSource objectSource, object t, object co, string name)
+		public virtual ValueReference GetMember (EvaluationContext ctx, IObjectSource objectSource, object t, object co, string name)
```

Comment:
If you make this method virtual, the ParentSource assignment below won't be done if this member is overriden and base is not called. So please don't make this method virtual and create a new protected one instead that is called here.